woocommerce/woocommerce-ios
# Jetpack connection: Go to connection step if `connectionOnly` is true

Closes: #10922

## Description

The retrieve Jetpack plugin details endpoint `wp-json/wp/v2/plugins/jetpack/jetpack` returns 403 if we access it using a shop manager account. 

As a solution, we are skipping to the Jetpack connection step directly if the view model's `connectionOnly` value is true. 

## Testing instructions

1. Create a new JN site.
2. Install and connect Jetpack using your primary WPCOM account.
3. Create a new "Shop Manager" account from the `wp-admin` page. The email is from a different WPCOM account named `Alias`.
4. Login into the app using `Alias` WPCOM account. 
5. Go to the site picker and add the new JN site.
6. Enter the `wp-admin` credentials of the "Shop manager" account `Alias`.
7. You should be taken to the connect Jetpack step as the site already has JP installed.
8. Follow the steps to connect Jetpack.
9. You should land in the home screen after the Jetpack connection is complete. 

## Screenshots

https://github.com/woocommerce/woocommerce-ios/assets/524475/4486f284-a44d-477f-9d7f-cfc6a08ab4a3



---
